src/main/java/edu/tamu/scholars/middleware/config/model/ExportConfig.java

```diff
@@ -16,6 +19,8 @@ public class ExportConfig {
 
     private String individualBaseUri = "http://localhost:4200/display";
 
+    private Map<String, String> personTypeMapping = new HashMap<>();
```

Comment:
```suggestion
    private Map<String, String> personTypes = new HashMap<>();
```

I suggest renaming this to a shorter name, like `personTypes`.